webpack/assets/javascripts/react_app/components/common/ModalProgressBar/ModalProgressBar.scss

```diff
@@ -0,0 +1,17 @@
+#modal-progress-bar {
+  min-height: 100px;
+}
+
+// Needed to display modal within a container
+.modal-container {
+  position: relative;
+}
+
+.modal-container .modal,
+.modal-container .modal-backdrop {
+  position: absolute;
+}
+
+.modal-container.modal-open {
+  z-index: 0;
+}
```

Comment:
This file will be much more readable with `scss` syntax:
```scss
#modal-progress-bar {
  min-height: 100px;
}

 // Needed to display modal within a container
.modal-container {
  position: relative;

  .modal, .modal-backdrop {
    position: absolute;
  }

  &.modal-open {
    z-index: 0;
  }
}
```

Reply:
Fixed, thanks